packages/office-ui-fabric-react/src/components/TextField/TextField.tsx

```diff
@@ -270,6 +269,14 @@ export class TextField extends BaseComponent<ITextFieldProps, ITextFieldState> i
     }
   }
 
+  private _onRenderLabel(props: ITextFieldProps): JSX.Element | string | undefined {
+    let { label, onRenderLabel } = props;
+    return (onRenderLabel ?
```

Comment:
could say `onRenderLabel || label && (<Label htmlFor={ this._id }>{ onRenderLabel() || label }</Label>)` to reduce the <Label> duplication.